Code/MD_class.py

```diff
@@ -0,0 +1,76 @@
+import numpy as np
+from inspect import getsource
+
+
+class MD_Simulation_1D_model_system:
+
+    def __init__(self, total_steps, potential, derivative_potential, time_step, collision_rate, mass, temperature, kboltz=0.008314, initial_position=0, initial_velocity=0):
+        self.potential = potential
+        self.derivative_potential = derivative_potential
+        self.time_step = time_step
+        self.collision_rate = collision_rate
+        self.mass = mass
+        self.temperature = temperature
+        self.total_steps = total_steps
+        self.kboltz = kboltz
+        self.initial_position = initial_position
+        self.initial_velocity = initial_velocity
+        self.total_time = self.time_step * self.total_steps
+
+    def write_log_file(self, filepath):
+        f= open(filepath + "traj_LOG.txt","w+")
+        f.write("\n******* USED PARAMETERS ******** \n\n")
+        f.write("time step = %s \n" %self.time_step)
+        f.write("collision rate = %s \n" %self.collision_rate)
+        f.write("mass = %s \n" %self.mass)
+        f.write("temperature = %s \n" %self.temperature)
+        f.write("total number of steps = %s \n" %self.total_steps)
+        f.write("Boltzmann constant = %s \n" %self.kboltz)
+        f.write("initial position = %s \n" %self.initial_position)
+        f.write("initial velocity = %s \n" %self.initial_velocity)
+        f.write("%s" %getsource(self.potential))
+        f.write("%s" %getsource(self.derivative_potential))
+        f.close()
+
+    def draw_random_number_sequence(self, mean=0, variance=1, length=None):
+        if length == None:
+            eta = np.random.normal(mean, variance, (self.total_steps))
+        else:
+            eta = np.random.normal(mean, variance, (length))
+        return eta
+
```

Comment:
I think it would be good to rename these functions slighty: 
instead of overdamped_Langevin_dynamics and Langevin_dynamics there should be a function "compute_dynamics(self, integrator, ... ) " and then define the Integrator as either OLD or LD. With this change the functions will still be named after verb i.e. "do stuff" and might increase code readability.

Reply:
Done.